tableauserverclient/server/endpoint/groups_endpoint.py

```diff
@@ -25,15 +25,22 @@ def get(self, req_options=None):
     # Gets all users in a given group
     @api(version="2.0")
     def populate_users(self, group_item, req_options=None):
+        from .. import Pager
         if not group_item.id:
             error = "Group item missing ID. Group must be retrieved from server first."
             raise MissingRequiredFieldError(error)
+
+        # TODO should this be a list or a Pager directly?
+        all_users = list(Pager(lambda options: self._get_users_for_group(group_item, options), req_options))
```

Comment:
I really, really, really, really don't like turning this into a list.  Imagine a group that has 10,000 users.  This will do 100 calls to the server before returning. (or 10, depending on what the default page size is, I think it's 100, but I don't remember)

What if all you wanted to do was users[0]?  You'd be fetching way more than you need.

I'd rather break compatibility (since this does actually break backcompat because it doesn't return a pagination object) and called it iter_users() or something.